assets/sass/patterns/atoms/date.scss

```diff
@@ -6,6 +6,9 @@
 
   .teaser--main & {
     @include font-size(14);
+    @media only all and (max-width: 480px) {
```

Comment:
Can we use the variable for the width please.

Reply:
Ok. Can you confirm we are going with the 480px breakpoint?

Reply:
@Cosmin-Daniel See my comment at the bottom of the PR

Reply:
A variable is used now for the width.